tests/test_run_search.py

```diff
@@ -57,6 +57,105 @@ def setUp(self):
     def tearDown(self):
         self.mock_os.stop()
 
+
+    def test_full_sweep(self):
```

Comment:
Not sure whether to keep these tests or not. The tests in model_manager are a superset of these. Also, the tests in model_manager are better because they verify the full set of config combinations instead of just the expected count

Reply:
I would remove them. It makes for less to maintain

Reply:
removed